content/docs/glossary/claimable-balance.mdx

```diff
@@ -0,0 +1,75 @@
+   - Can claim if the ledger close time in which the claim was included is before X (Unix timestamp) - `CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)`
+   - Can claim if the ledger close time in which the claim was included is after X seconds + the ledger close time in which the ClaimableBalanceEntry was created - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_RELATIVE_TIME(X))`
+   - Can claim if the ledger close time in which the claim was included is at or after X (Unix timestamp) - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X))`
+   - Can claim between X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_AND(CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)), CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
+   - Can claim outside X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_OR(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X), CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
+
+### Operation Information
+
+This operation will move Amount of Asset from the operation source account into a new ClaimableBalanceEntry.
+
+Note that the baseReserve requirement for a ClaimableBalanceEntry is dependant on the number of Claimants. The [minimum balance](./minimum-balance.mdx) of the account will increase by `# of Claimants * baseReserve`.
+
+### BalanceID
+
+A successful Create Claimable Balance operation will return a balanceID, which is required to claim the ClaimableBalanceEntry using the Claim Claimable Balance operation. The balanceID is the SHA-256 hash of the OperationID (see [stellar-transactions.x](https://github.com/stellar/stellar-core/blob/master/src/xdr/Stellar-transaction.x)), which contains the transaction source account, sequence number, and the operation index of the CreateClaimableBalance operation in this transaction.
+
+
+[Claim Claimable Balance](../start/list-of-operations.mdx#claim-claimable-balance)
+
+### Parameters
+
+1. BalanceID - The ID of the ClaimableBalanceEntry being claimed.
+
+### Operation Information
+
+This operation will load the ClaimableBalanceEntry that corresponds to the BalanceID, and then search for the source account of this operation in the list of Claimants on the entry. If a match on the Claimant is found, and the ClaimPredicate evaluates to true, then the ClaimableBalanceEntry can be claimed. The balance on the entry will be moved to the source account if there are no limit or trustline issues(for non-native assets).
+
+Once a ClaimableBalanceEntry has been claimed, it will be deleted.
+
+## Example
+
+The below pseudocode details how AccountA can create a ClaimableBalanceEntry with two Claimants, AccountA (itself) and AccountB, that can claim the balance under different conditions.
+<CodeExample>
+
+```cpp
+// In this scenario, AccountB has not been created yet, but we know the public key
+Claimants = [Claimant(AccountA, CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(100))), Claimant(AccountB, CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(50))]
```

Comment:
Just caught this: `T=100` for absolute time here doesn't make sense since that'd be 100 seconds into Jan 1st 1970, heh.

Reply:
I thought it would be easier to read with smaller numbers, but I see how this could confuse the reader. I think this section can be removed anyways in favor of SDK examples.